handlers/main.yml

```diff
@@ -4,10 +4,22 @@
     name: nginx
     state: reloaded
 
+- name: restart nginx
+  service:
+    name: nginx
+    state: restarted
+
 - name: restart bigbluebutton
   command: bbb-conf --restart
   become: true
 
+- name: restart freeswitch
+  systemd:
+    name: freeswitch
+    state: restarted
+    daemon_reload: yes
```

Comment:
Does it need to reload systemd ?

Reply:
It seems so according to BBB documentation: 
https://docs.bigbluebutton.org/2.2/troubleshooting.html#freeswitch-fails-to-bind-to-port-8021

Reply:
Jes, indeed. This is needed, because `/lib/systemd/system/freeswitch.service` gets edited. But then only as a handler for `disable CPUSchedulingPolicy in FreeSWITCH service`

Reply:
This has been now fixed in: https://github.com/n0emis/ansible-role-bigbluebutton/pull/3/commits/669fe9562e847ccfd9f05173a74465a458fb46a2